components/ModEditSubmissionTopbar.vue

```diff
@@ -48,12 +48,14 @@
 
 <script setup lang="ts">
 import { ref, type Ref } from 'vue'
+import { gql } from 'graphql-request'
 import SVGCopyContent from '~/assets/icons/content-copy.svg'
 import SVGSuccessCheckMark from '~/assets/icons/checkmark-square.svg'
 import { useModerationSubmissionsStore } from '~/stores/moderationSubmissionsStore'
+import { gqlClient } from '#imports'
```

Comment:
let's use an explicit path so we know where it's coming from. It will be easier for devs reading to understand where it comes from